src/players/VAST.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import videojs from 'maestro-videojs-vast';
+import 'maestro-videojs-vast/dist/maestroVideoJsVast.css';
+import { callPlayer } from '../utils';
+import createSinglePlayer from '../singlePlayer';
+
+const PLACEHOLDER_VIDEO = 'https://storage.googleapis.com/maestro_video/blank.mp4';
+export class VAST extends Component {
+  static displayName = 'VAST';
+
+  static canPlay = url => typeof url === 'string';
```

Comment:
going to need a more specific one here

Reply:
the URL regex in this package doesn't allow any VAST urls pass, and there's not standard VAST url format.

Reply:
i think you are on an out of date master unfortunately. our iframe player is just `return true`...